app/helpers/breadcrumbs_organization_helper.rb

```diff
@@ -0,0 +1,20 @@
+module BreadcrumbsOrganizationHelper
+  def organization_breadcrumb(has_link)
+    breadcrumb = Organization.current.breadcrumb_image_url.present? ? image_breadcrumb : text_breadcrumb
```

Comment:
This line could be extracted to a method and hence this variable could be ommited.

Reply:
Done.